node/subsystem-util/src/runtime/mod.rs

```diff
@@ -73,32 +86,47 @@ pub struct ValidatorInfo {
 	pub our_group: Option<GroupIndex>,
 }
 
+impl Default for Config {
+	fn default() -> Self {
+		Self {
+			keystore: None,
+			session_cache_lru_size: 2,
```

Comment:
The old one had some rationale to why `2` is a good idea, I'd prefer to keep that.